src/frontend/Komponenter/Oppgavebenk/typer/enhet.ts

```diff
@@ -7,24 +7,33 @@ export enum FortroligEnhet {
     VIKAFOSSEN = '2103',
 }
 
-export const enhetTilTekstIkkeFortrolig: Record<IkkeFortroligEnhet, string> = {
+const enhetTilTekstIkkeFortrolig: Record<IkkeFortroligEnhet.NAY, string> = {
     '4489': '4489 NAY',
+};
+
+const enhetTilTekstEgenAnsatte: Record<IkkeFortroligEnhet.EGNE_ANSATTE, string> = {
     '4483': '4483 Egne ansatte',
 };
 
-export const enhetTilTekstFortrolig: Record<FortroligEnhet, string> = {
+const enhetTilTekstFortrolig: Record<FortroligEnhet, string> = {
     '2103': '2103 NAV Vikafossen',
 };
 
 export const enhetTilTekstPåString: Record<string, string> = {
     ...enhetTilTekstIkkeFortrolig,
     ...enhetTilTekstFortrolig,
+    ...enhetTilTekstEgenAnsatte,
 };
 
 export const enhetTilTekst = (
-    harSaksbehandlerStrengtFortroligRolle: boolean
+    harSaksbehandlerStrengtFortroligRolle: boolean,
+    harSaksbehandlerEgenAnsattRolle: boolean
 ): Record<string, string> => {
-    return harSaksbehandlerStrengtFortroligRolle
-        ? enhetTilTekstFortrolig
-        : enhetTilTekstIkkeFortrolig;
+    if (harSaksbehandlerStrengtFortroligRolle) {
+        return enhetTilTekstFortrolig;
+    } else if (harSaksbehandlerEgenAnsattRolle) {
```

Comment:
Her har du mistet "vanlig fortrolig" - er de også 4483 eller er de 4489 ? 🤔

Reply:
Mirja var enig om at egne ansatte ikke skal vises for de som ikke har tilgang ✅

Reply:
Skjønner ikke helt hva du tenker på.
Tanken nå var å bare fjerne egenAnsatt for de som ikke har den rollen, som nå gjøres.

Når jeg skrev "fortrolig" i forrige posten, så mener jeg strengt fortrolig, som er likt som tidligere.

Reply:
Mulig det ble forvirring fordi jeg skrev fortrolig i forrige post, og ikke strengt fortrolig.
Eller så skjønner jeg ikke hva du tenker på. Ta gjerne en titt på kodeendringene på nytt.

Tanken nå var å bare fjerne egenAnsatt for de som ikke har den rollen, som nå gjøres.
Fortrolig blir håndtert som tidligere, de får 4489, og 4483 hvis de også har rollen for egen ansatt.

Blir fort sånt når koden er "fortrolig" men når det egentlige gjelder strengt fortroligt:
```js
const enhetTilTekstFortrolig: Record<FortroligEnhet, string> = {
    '2103': '2103 NAV Vikafossen',
};
```
Burde kanskje endres til
```js
const enhetTilTekstStrengtFortrolig: Record<StrengtFortroligEnhet, string> = {
    '2103': '2103 NAV Vikafossen',
};
```

Reply:
Ble litt forvirring her pga at vi snakket om 2 ulike saker 😂 
Jeg snakket om hva man skulle se, Charlie snakket om hvor oppgaver for kode 7 havner